Neos.Flow/Classes/Http/BaseUriProvider.php

```diff
@@ -0,0 +1,88 @@
+<?php
+namespace Neos\Flow\Http;
+
+use GuzzleHttp\Psr7\Uri;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Http\Helper\RequestInformationHelper;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * Supports to get a baseUri from various possible sources.
+ *
+ * @Flow\Scope("singleton")
+ */
+class BaseUriProvider
+{
+    /**
+     * THe possibly configured Flow base URI.
+     *
+     * @Flow\InjectConfiguration(package="Neos.Flow", path="http.baseUri")
+     * @var string|null
+     */
+    protected $configuredBaseUri;
+
+    /**
+     * @Flow\Inject
+     * @var Bootstrap
+     */
+    protected $bootstrap;
+    /**
```

Comment:
```suggestion

    /**
``` (nitpick)

Reply:
I don't get the suggesion^^

Reply:
Add an empty line after `protected $bootstrap;`